src/sentry/integrations/jira/client.py

```diff
@@ -189,18 +189,17 @@ def get_users_for_project(self, project):
 
     def search_users_for_project(self, project, username):
         # Jira Server wants a project key, while cloud is indifferent.
+        # Use the query parameter as our implemention follows jira's gdpr practices
         project_key = self.get_project_key_for_id(project)
         return self.get_cached(
             self.USERS_URL,
-            params={'project': project_key, 'username': username})
+            params={'project': project_key, 'query': username})
```

Comment:
Is there a chance that this will return multiple users?

Reply:
Very possible and desirable. The results of this method are fed into autocomplete widgets in the ui.